cds_ils/importer/providers/ebl/ebl.py

```diff
@@ -0,0 +1,10 @@
+from cds_ils.importer.base import Base
+
+
+class EBLModel(Base):
+
+    __query__ = ''
+
+    __schema__ = 'https://127.0.0.1:5000/schemas/documents/document-v1.0.0.json'
```

Comment:
```suggestion
    __schema__ = 'https://127.0.0.1:5000/schemas/documents/document-v1.0.0.json'
```
watch out to the HOST here, it is different when deployed. Please use the config `JSONSCHEMA_HOST`. You can use utilities if needed, see [here](https://github.com/inveniosoftware/invenio-jsonschemas/blob/master/invenio_jsonschemas/ext.py#L138)

Reply:
the json schema host is solved by dojson, I just didn't clean this part yet